python/ml4ir/base/model/relevance_model.py

```diff
@@ -416,14 +418,18 @@ def fit(
                     to differentiate from validation and test metrics
                     in the final experiment results
                     """
-                    train_metrics["train_{}".format(metric)] = value[0]
+                    # History is a dict of key: list(values per epoch)
+                    # We are capturing the metrics of the last epoch (-1)
```

Comment:
`history.history` looks like 
```
{"categorical_accuracy": [1.80, 1.65, 1.60, ...]} 
```
where the dim of the values is the number of epochs. The previous version uses the value of the first epoch. I find this misleading when reporting results. I am changing the behavior to get the info of the last epoch. 

To be honest, I would prefer returning the actual history here so that the user has the complete information. `train_metrics` is a subset of information and lots of interesting insights are missing.